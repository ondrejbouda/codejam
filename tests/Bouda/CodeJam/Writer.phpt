<?php

namespace BoudaTests;

use Bouda;
use Tester;
use Tester\Assert;

require_once __DIR__ . '/../../bootstrap.php';


$outputFile = Tester\FileMock::create('');
$outputFileExpected = 'Writer.out.expected';

$writer = new Bouda\CodeJam\Writer($outputFile);

$writer->writeLine('line 1');
$writer->write('part1 ');
$writer->write('part2 ');
$writer->newline();
$writer->writeLine('line 3');

$writer->close();

Assert::matchFile($outputFileExpected, file_get_contents($outputFile));
