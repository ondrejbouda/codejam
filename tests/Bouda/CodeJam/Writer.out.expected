line 1
part1 part2
line 3
